### ISSUE TITLE
query params in POST form are not sent to the server

### ISSUE THREAD

**Issue by nshafer** [unpoly/unpoly#77]
If you have a form with query parameters in the action, they are not sent as part of the posted URL.

```
<form action="/some/url?foo=bar" method="post" up-target=".container">
```

Normally this is perfectly valid, and on the server side both the query string is sent with the post, and the post variables are available in the body, but not when using `up-target`.

Inspecting this in devtools or on the server shows that the query parameter is stripped from the URL before it does the POST. So I would expect to see `POST /some/url/foo=bar HTTP/1.1` on the server side, but with the up-target included, all I get is `POST /some/url HTTP/1.1`.

Thanks!
Nate

**Comment by triskweline** [comment]
Params in the query string are currently moved to the request body for non-GET requests.

This behavior should be changed.

**Comment by OvermindDL1** [comment]
Oh, this explains why my filter form never worked with unpoly, didn't look too close in to it and thought it was my bug...

**Comment by sturatcliffe** [comment]
@triskweline Any suggestions on a workaround for this one? We're currently using version 0.51.1 as we haven't updated for a while, and this issue isn't present in that version. However, as ASP.NET core makes use of POST action URL's with query params in, it's blocking us from updating to the latest version.

I'd imagine a compiler to manually call up.submit might work? Unless you have a better suggestion? Cheers!

**Comment by nshafer** [comment]
My workaround for now has just been to not use `up-target` on that form. I'm only using unpoly as a progressive enhancement, so it's not too bad to do a full refresh in this one part of my app that's not used a whole lot.

**Comment by triskweline** [comment]
Fixed in 0.60.2.
